protoc_plugin/test/goldens/grpc_service.pbgrpc

```diff
@@ -30,6 +30,18 @@ class TestClient extends $grpc.Client {
       '/Test/Bidirectional',
       ($0.Input value) => value.writeToBuffer(),
       ($core.List<$core.int> value) => $0.Output.fromBuffer(value));
+  static final _$new_ = $grpc.ClientMethod<$0.Input, $0.Output>(
```

Comment:
One thing I was curious of, is it possible to have a name collision with the library import aliases? i.e. do we also need to add `r'$grpc'`, `r'$async'` etc to the reserved words?

Reply:
$ cannot occur in a protobuf identifier, (https://developers.google.com/protocol-buffers/docs/reference/proto3-spec#identifiers) - I also tried it out and the protoc compiler gave a syntax error.

I believe that is why we chose to use $ in the first place for these names.

That means we could actually remove all the reserved words with $!

I won't do that in this PR to keep the scope smaller.

Reply:
(I removed it from those added).